zebra-rpc/src/methods/tests/snapshots/get_blockchain_info@testnet_10.snap

```diff
@@ -37,11 +37,6 @@ expression: info
       "name": "NU5",
       "activationheight": 1842420,
       "status": "pending"
-    },
-    "c8e71055": {
-      "name": "Nu6",
-      "activationheight": 2942000,
-      "status": "pending"
```

Comment:
Do we want to do the same as the getblocksubsidy and have a snapshot with this included ?

Reply:
Yeah, we'll need it shortly, added in https://github.com/ZcashFoundation/zebra/pull/8766/commits/4dba1e8746cbcd93e4f660927052487b68347184